misk-hibernate/src/test/kotlin/misk/hibernate/ShardIdGenerator.kt

```diff
@@ -0,0 +1,16 @@
+package misk.hibernate
+
+class ShardIdGenerator constructor(private val shard: Shard) {
```

Comment:
where is this used?

Reply:
no where, was using it in testing. Removed it.